Tasks/ANT/ant.ps1

```diff
@@ -187,7 +187,7 @@ if($publishJUnitResultsFromAntBuild)
 	$runTitleMemberExists = CmdletHasMember "RunTitle"
 	if($runTitleMemberExists)
 	{
-		Publish-TestResults -TestRunner "JUnit" -TestResultsFiles $matchingTestResultsFiles -Context $distributedTaskContext -RunTitle $testRunTitle
+		Publish-TestResults -TestRunner "JUnit" -TestResultsFiles $matchingTestResultsFiles -Context $distributedTaskContext -RunTitle $testRunTitle -MergeResults $true
```

Comment:
I would recommend merging by default, when the task was originally written merge was not an option on xplat, so we default to merge false but for performance users would want to merge in most cases.